Kernel fault error with Raspberry PI 3 B (kernel version 4.19.127)

[WLAN] RTK Loading modules...
[   25.968575] 8192eu: loading out-of-tree module taints kernel.
[   26.312268] RTW: module init start
[   26.312746] RTW: rtl8192eu v5.6.4_35685.20191108_COEX20171113-0047
[   26.313616] RTW: build time: Jul 15 2020 13:01:24
[   26.314268] RTW: rtl8192eu BT-Coex version = COEX20171113-0047
[   26.315140] RTW: rtw_inetaddr_notifier_register
[   26.315955] RTW:
[   26.315955] usb_endpoint_descriptor(0):
[   26.316763] RTW: bLength=7
[   26.317154] RTW: bDescriptorType=5
[   26.317610] RTW: bEndpointAddress=84
[   26.318128] RTW: wMaxPacketSize=512
[   26.318605] RTW: bInterval=0
[   26.319028] RTW: RT_usb_endpoint_is_bulk_in = 4
[   26.319639] RTW:
[   26.319639] usb_endpoint_descriptor(1):
[   26.320442] RTW: bLength=7
[   26.320808] RTW: bDescriptorType=5
[   26.321297] RTW: bEndpointAddress=5
[   26.321771] RTW: wMaxPacketSize=512
[   26.322266] RTW: bInterval=0
[   26.322662] RTW: RT_usb_endpoint_is_bulk_out = 5
[   26.323311] RTW:
[   26.323311] usb_endpoint_descriptor(2):
[   26.324122] RTW: bLength=7
[   26.324493] RTW: bDescriptorType=5
[   26.324953] RTW: bEndpointAddress=6
[   26.325457] RTW: wMaxPacketSize=512
[   26.325933] RTW: bInterval=0
[   26.326350] RTW: RT_usb_endpoint_is_bulk_out = 6
[   26.326974] RTW:
[   26.326974] usb_endpoint_descriptor(3):
[   26.327771] RTW: bLength=7
[   26.328159] RTW: bDescriptorType=5
[   26.328625] RTW: bEndpointAddress=87
[   26.329138] RTW: wMaxPacketSize=64
[   26.329600] RTW: bInterval=3
[   26.330014] RTW: RT_usb_endpoint_is_int_in = 7, Interval = 3
[   26.330781] RTW:
[   26.330781] usb_endpoint_descriptor(4):
[   26.331597] RTW: bLength=7
[   26.331969] RTW: bDescriptorType=5
[   26.332457] RTW: bEndpointAddress=8
[   26.332930] RTW: wMaxPacketSize=512
[   26.333429] RTW: bInterval=0
[   26.333818] RTW: RT_usb_endpoint_is_bulk_out = 8
[   26.334468] RTW: nr_endpoint=5, in_num=2, out_num=3
[   26.334468]
[   26.335335] RTW: USB_SPEED_HIGH
[   26.335759] RTW: CHIP TYPE: RTL8192E
[   26.336622] RTW: read_chip_version_8192e 0xF0 = 0xc441135
[   26.337375] RTW: rtw_hal_config_rftype RF_Type is 2 TotalTxPath is 2
[   26.338259] RTW: Chip Version Info: CHIP_8192E_Normal_Chip_SMIC_B_CUT_2T2R_RomVer(0)
[   26.339311] RTW: _ConfigChipOutEP_8192E OutEpQueueSel(0x07), OutEpNumber(3)
[   26.340492] RTW: Boot from EFUSE, Autoload OK !
[   26.477199] RTW: HW EFUSE
[   26.477569] RTW: 0x000: 29 81 00 7C  01 40 03 00  40 74 04 50  14 00 00 00
[   26.478592] RTW: 0x010: 27 27 26 26  26 26 2A 2A  29 29 29 F2  EF EF FF FF
[   26.479597] RTW: 0x020: FF FF FF FF  FF FF FF FF  FF FF FF FF  FF FF FF FF
[   26.480610] RTW: 0x030: FF FF FF FF  FF FF FF FF  FF FF 2A 2A  28 28 28 28
[   26.481610] RTW: 0x040: 2B 2B 2A 29  29 F2 EF EF  FF FF FF FF  FF FF FF FF
[   26.482624] RTW: 0x050: FF FF FF FF  FF FF FF FF  FF FF FF FF  FF FF FF FF
[   26.483645] RTW: 0x060: FF FF FF FF  FF FF FF FF  FF FF FF FF  FF FF FF FF
[   26.484657] RTW: 0x070: FF FF FF FF  FF FF FF FF  FF FF FF FF  FF FF FF FF
[   26.485655] RTW: 0x080: FF FF FF FF  FF FF FF FF  FF FF FF FF  FF FF FF FF
[   26.486669] RTW: 0x090: FF FF FF FF  FF FF FF FF  FF FF FF FF  FF FF FF FF
[   26.487666] RTW: 0x0A0: FF FF FF FF  FF FF FF FF  FF FF FF FF  FF FF FF FF
[   26.488675] RTW: 0x0B0: FF FF FF FF  FF FF FF FF  A2 1F 20 00  00 00 FF FF
[   26.489673] RTW: 0x0C0: FF 01 00 10  00 00 00 FF  00 00 FF FF  FF FF FF FF
[   26.490684] RTW: 0x0D0: 57 23 09 01  E7 47 02 D4  6E 0E 13 B9  8C 0A 03 52
[   26.491700] RTW: 0x0E0: 65 61 6C 74  65 6B 20 0E  03 38 30 32  2E 31 31 6E
[   26.492713] RTW: 0x0F0: 20 4E 49 43  20 00 00 FF  FF FF FF FF  FF FF FF FF
[   26.493711] RTW: 0x100: FF FF FF FF  FF FF FF FF  FF FF FF FF  FF FF FF FF
[   26.494723] RTW: 0x110: FF FF FF FF  FF FF FF 0D  03 00 05 00  30 00 00 00
[   26.495721] RTW: 0x120: 00 93 FF FF  FF FF FF FF  FF FF FF FF  FF FF FF FF
[   26.496730] RTW: 0x130: FF FF FF FF  FF FF FF FF  FF FF FF FF  FF FF FF FF
[   26.497728] RTW: 0x140: FF FF FF FF  FF FF FF FF  FF FF FF FF  FF FF FF FF
[   26.498739] RTW: 0x150: FF FF FF FF  FF FF FF FF  FF FF FF FF  FF FF FF FF
[   26.499762] RTW: 0x160: FF FF FF FF  FF FF FF FF  FF FF FF FF  FF FF FF FF
[   26.500773] RTW: 0x170: FF FF FF FF  FF FF FF FF  FF FF FF FF  FF FF FF FF
[   26.501771] RTW: 0x180: FF FF FF FF  FF FF FF FF  FF FF FF FF  FF FF FF FF
[   26.502784] RTW: 0x190: FF FF FF FF  FF FF FF FF  FF FF FF FF  FF FF FF FF
[   26.503784] RTW: 0x1A0: FF FF FF FF  FF FF FF FF  FF FF FF FF  FF FF FF FF
[   26.504792] RTW: 0x1B0: FF FF FF FF  FF FF FF FF  FF FF FF FF  FF FF FF FF
[   26.505789] RTW: 0x1C0: FF FF FF FF  FF FF FF FF  FF FF FF FF  FF FF FF FF
[   26.506801] RTW: 0x1D0: FF FF FF FF  FF FF FF FF  FF FF FF FF  FF FF FF FF
[   26.507819] RTW: 0x1E0: FF FF FF FF  FF FF FF FF  FF FF FF FF  FF FF FF FF
[   26.508831] RTW: 0x1F0: FF FF FF FF  FF FF FF FF  FF FF FF FF  FF FF FF FF
[   26.509833] RTW: EEPROM ID=0x8129
[   26.510313] RTW: VID = 0x2357, PID = 0x0109
[   26.510881] RTW: Customer ID: 0x00, SubCustomer ID: 0xCD
[   26.511626] RTW: Hal_ReadPowerSavingMode8192E...bHWPwrPindetect(0)-bHWPowerdown(0) ,bSupportRemoteWakeup(1)
[   26.512936] RTW: ### PS params=>  power_mgnt(2),usbss_enable(0) ###
[   26.513813] RTW: EEPROMRegulatory = 0x1
[   26.514349] RTW: Board Type: Dongle or WIFI only Module
[   26.515080] RTW: Hal_EfuseParseBTCoexistInfo8192E: Disable BT-coex, wifi ant_num=2
[   26.516126] RTW: hal_com_config_channel_plan chplan:0x22
[   26.516838] RTW: crystal_cap: 0x1f
[   26.517330] RTW: ThermalMeter = 0x20
[   26.517818] RTW: SWAS: bHwAntDiv = 0, TRxAntDivType = 0
[   26.518546] RTW: pHalData->ExternalPA_2G = 0 , pHalData->ExternalLNA_2G = 0
[   26.519490] RTW: pHalData->TypeGLNA is 0x0
[   26.520071] RTW: pHalData->rfe_type is 0xff
[   26.522653] RTW: kfree flag:0
[   26.523080] RTW: rtw_hal_read_chip_info in 183 ms
[   26.523751] RTW: init_channel_set((null)) ChannelPlan ID:0x22, ch num:11
[   26.524861] RTW: init_mlme_default_rate_set: support CCK
[   26.525592] RTW: init_mlme_default_rate_set: support OFDM
[   26.527053] RTW: NR_RECVBUFF: 8
[   26.527484] RTW: MAX_RECVBUF_SZ: 32768
[   26.528031] RTW: NR_PREALLOC_RECV_SKB: 8
[   26.528943] RTW: rtw_alloc_macid((null)) if1, mac_addr:ff:ff:ff:ff:ff:ff macid:1
[   26.530001] RTW: IQK FW offload:disable
[   26.530532] RTW: rtw_regsty_chk_target_tx_power_valid return _FALSE for band:0, path:0, rs:0, t:-1
[   26.531782] RTW: pwrctrlpriv.bSupportRemoteWakeup~~~~~~
[   26.532500] RTW: pwrctrlpriv.bSupportRemoteWakeup~~~[1]~~~
[   26.533267] RTW: can't get autopm:
[   26.533737] RTW: rtw_macaddr_cfg mac addr:d4:6e:0e:13:b9:8c
[   26.534513] RTW: bDriverStopped:True, bSurpriseRemoved:False, bup:0, hw_init_completed:0
[   26.535664] RTW: rtw_wiphy_alloc(phy1)
[   26.536207] RTW: rtw_wdev_alloc(padapter=0000000024465bc4)
[   26.536950] RTW: rtw_wiphy_register(phy1)
[   26.537519] RTW: Register RTW cfg80211 vendor cmd(0x67) interface
[   26.538547] RTW: rtw_reg_notifier: NL80211_REGDOM_SET_BY_USER alpha2:US
[   26.539464] RTW: rtw_set_country(): not applied
[   26.544820] RTW: rtw_ndev_init(wlan1) if1 mac_addr=d4:6e:0e:13:b9:8c
[   26.546192] RTW: rtw_ndev_notifier_call(wlan1) state:16
[   26.548079] RTW: cfg80211_rtw_get_txpower
[   26.548638] RTW: rtw_ndev_notifier_call(wlan1) state:5
[   26.549634] usbcore: registered new interface driver rtl8192eu
[   26.550450] RTW: module init ret=0
[   26.569030] RTW: rtw_ndev_notifier_call(wlan1) state:13
[   26.569749] RTW: _netdev_open(wlan1) , bup=0
[   26.570372] RTW: ==>_InitPowerOn_8192EU
[   26.574243] RTW: SetHwReg: bMacPwrCtrlOn=1
[   26.575471] RTW:  ===> FirmwareDownload88E() fw source from Header.
[   26.576333] RTW: FirmwareDownload8192E fw:NIC, size: 32286
[   26.577106] RTW: FirmwareDownload8192E: fw_ver=35 fw_subver=7 sig=0x92e1
[   26.603561] RTW: polling_fwdl_chksum: Checksum report OK! (1, 0ms), REG_MCUFWDL:0x00070305
[   26.606386] RTW: =====> _8051Reset8192E(): 8051 reset success .
[   26.634702] RTW: _FWFreeToGo8192E: Polling FW ready OK! (245, 30ms), REG_MCUFWDL:0x000706c6
[   26.635859] RTW: FWDL success. write_fw:1, 57ms
[   27.251799] RTW: Auto InitLLTTable8192E success
[   27.419137] RTW: phy_SpurCalibration_8192E =>AFE_PHASE_SEL
[   27.484110] RTW: ===FixSpur NOT Pass!
[   27.883945] RTW: rtl8192eu_hal_init in 1313ms
[   27.884564] RTW: rtw_hal_set_macaddr_port wlan1- hw port(0) mac_addr =d4:6e:0e:13:b9:8c
[   27.887043] RTW: rtw_hal_get_macaddr_port wlan1- hw port(0) mac_addr =d4:6e:0e:13:b9:8c
[   27.888138] RTW: wlan1- hw port(0) mac_addr =d4:6e:0e:13:b9:8c
[   27.889296] RTW: [HW_VAR_ENABLE_RX_BAR] 0x6A2=0x100
[   27.907004] RTW: wlan1- Port-0  set opmode = 2
[   27.907629] RTW: rtw_hal_set_macaddr_port wlan1- hw port(0) mac_addr =d4:6e:0e:13:b9:8c
[   27.910442] RTW: MAC Address = d4:6e:0e:13:b9:8c
[   27.911090] RTW: rtw_start_drv_threads(wlan1) enter
[   27.911747] RTW: rtw_start_drv_threads(wlan1) start RTW_CMD_THREAD
[   27.912958] RTW: rtw_cfg80211_init_wiphy:rf_type=2
[   27.913629] RTW: [HT] HAL Support STBC = 0x01
[   27.914248] RTW: -871x_drv - drv_open, bup=1
[   27.915162] IPv6: ADDRCONF(NETDEV_UP): wlan1: link is not ready
[   27.916016] RTW: cfg80211_rtw_set_power_mgmt(wlan1) enabled:1, timeout:-1
[   27.916927] RTW: rtw_ndev_notifier_call(wlan1) state:1
[WLAN] Building up /var/tmp/wpa_supplicant/1.conf
[WLAN] Running WPA Suppplicant !!
Successfully initialized wpa_supplicant
[   28.085243] RTW: [rtw_wx_set_pmkid] IW_PMKSA_FLUSH!
[   28.085935] RTW: rtw_ps_deny_cancel(wlan1): [ERROR] Reason 7 had been canceled before!!
[   28.087047] RTW: set_mode = IW_MODE_INFRA
[   28.087634] RTW: wlan1- Port-0  set opmode = 2
[   28.088285] RTW: rtw_hal_set_macaddr_port wlan1- hw port(0) mac_addr =d4:6e:0e:13:b9:8c
[   28.092064] RTW: set bssid:00:00:00:00:00:00
ioctl[SI[   28.092722] RTW: =>rtw_wx_set_essid
OCSIWAP][   28.093319] RTW: <=rtw_wx_set_essid, ret 0
: Operation not permitted
[   29.109514] RTW: rtw_ndev_notifier_call(wlan1) state:4
[   29.120211] RTW: [rtw_wx_set_pmkid] IW_PMKSA_FLUSH!
[   29.247141] RTW: rtw_wx_set_scan(wlan1)
[   29.248253] RTW: wlan1 sleep m0=0x00000002, ori reg_0x4d4=0x00000000
udhcpc: started, v1.28.0
Waiting IP .. 0
Setting IP address 0.0.0.0 on wlan1
udhcpc: sending discover
[   29.922547] RTW: _rtw_phydm_pwr_tracking_rate_check i=0 tx_rate =0x2
[   29.923433] RTW: _rtw_phydm_pwr_tracking_rate_check tx_low_rate (unlinked to any AP)=0x2
Waiting IP .. 1
[   30.598461] RTW: wlan1 wakeup m0=0x00000002, ori reg_0x4d4=0x00000002
[   30.599514] RTW: survey done event(33) band:0 for wlan1
[   30.600368] RTW: rtw_indicate_scan_done(wlan1)
[   30.602019] RTW: ==>rtw_ps_processor .fw_state(8)
[   30.602654] RTW: ==>ips_enter cnts:1
[   30.603169] RTW: nolinked power save enter
[   30.603719] RTW: ===> rtw_ips_pwr_down...................
[   30.604473] RTW: ====> rtw_ips_dev_unload...
[   30.632057] RTW: usb_read_port_cancel
[   30.632572] RTW: usb_read_port_complete() RX Warning! bDriverStopped(False) OR bSurpriseRemoved(False)
[   30.633848] RTW: usb_read_port_complete() RX Warning! bDriverStopped(False) OR bSurpriseRemoved(False)
[   30.635124] RTW: usb_read_port_complete() RX Warning! bDriverStopped(False) OR bSurpriseRemoved(False)
[   30.636419] RTW: usb_read_port_complete() RX Warning! bDriverStopped(False) OR bSurpriseRemoved(False)
[   30.637717] RTW: usb_read_port_complete() RX Warning! bDriverStopped(False) OR bSurpriseRemoved(False)
[   30.639008] RTW: usb_read_port_complete() RX Warning! bDriverStopped(False) OR bSurpriseRemoved(False)
[   30.640284] RTW: usb_read_port_complete() RX Warning! bDriverStopped(False) OR bSurpriseRemoved(False)
[   30.641558] RTW: usb_read_port_complete() RX Warning! bDriverStopped(False) OR bSurpriseRemoved(False)
[   30.642823] RTW: usb_write_port_cancel
[   30.643391] RTW: ==> rtl8192eu_hal_deinit
[   30.644635] RTW: bkeepfwalive(0)
[   30.645100] RTW:  hal_poweroff_8192eu
[   30.649432] RTW: SetHwReg: bMacPwrCtrlOn=0
[   30.650009] RTW: <=== rtw_ips_pwr_down..................... in 47ms
...
...
[   39.253477] RTW: _rtw_pwr_wakeup call ips_leave....
[   39.254173] RTW: ==>ips_leave cnts:1
[   39.254657] RTW: ===>  rtw_ips_pwr_up..............
[   39.255361] RTW: ===> ips_netdrv_open.........
[   39.255968] RTW: ==>_InitPowerOn_8192EU
[   39.259809] RTW: SetHwReg: bMacPwrCtrlOn=1
[   39.260947] RTW:  ===> FirmwareDownload88E() fw source from Header.
[   39.261822] RTW: FirmwareDownload8192E fw:NIC, size: 32286
[   39.262582] RTW: FirmwareDownload8192E: fw_ver=35 fw_subver=7 sig=0x92e1
[   39.289190] RTW: polling_fwdl_chksum: Checksum report OK! (1, 0ms), REG_MCUFWDL:0x00070605
[   39.292005] RTW: =====> _8051Reset8192E(): 8051 reset success .
[   39.320363] RTW: _FWFreeToGo8192E: Polling FW ready OK! (243, 30ms), REG_MCUFWDL:0x000706c6
[   39.321502] RTW: FWDL success. write_fw:1, 57ms
[   39.938797] RTW: Auto InitLLTTable8192E success
[   40.101796] RTW: phy_SpurCalibration_8192E =>AFE_PHASE_SEL
[   40.166112] RTW: === PathA=1, PathB=1
[   40.166611] RTW: ===FixSpur Pass!
[   40.188464] RTW: rtl8192eu_hal_init in 933ms
[   40.189077] RTW: rtw_hal_set_macaddr_port wlan1- hw port(0) mac_addr =d4:6e:0e:13:b9:8c
[   40.191544] RTW: rtw_hal_get_macaddr_port wlan1- hw port(0) mac_addr =d4:6e:0e:13:b9:8c
[   40.192643] RTW: wlan1- hw port(0) mac_addr =d4:6e:0e:13:b9:8c
[   40.193816] RTW: [HW_VAR_ENABLE_RX_BAR] 0x6A2=0x100
[   40.211567] RTW: wlan1- Port-0  set opmode = 2
[   40.212200] RTW: rtw_hal_set_macaddr_port wlan1- hw port(0) mac_addr =d4:6e:0e:13:b9:8c
[   40.215122] RTW: <===  rtw_ips_pwr_up.............. in 961ms
[   40.215882] RTW: nolinked power save leave
[   40.216626] RTW: ==> ips_leave.....LED(0x0f628282)...
[   40.217336] RTW: =>rtw_wx_set_essid
[   40.217813] RTW: <=rtw_wx_set_essid, ret 0
[   40.218530] RTW: wpa_set_auth_algs, AUTH_ALG_OPEN_SYSTEM
[   40.219285] RTW: rtw_ps_deny_cancel(wlan1): [ERROR] Reason 7 had been canceled before!!
[   40.220383] RTW: set_mode = IW_MODE_INFRA
[   40.220966] RTW: wlan1- Port-0  set opmode = 2
[   40.221600] RTW: rtw_hal_set_macaddr_port wlan1- hw port(0) mac_addr =d4:6e:0e:13:b9:8c
[   40.225230] RTW:
[   40.225230]  wpa_ie(length:22):
[   40.225919] RTW: 0x30 0x14 0x01 0x00 0x00 0x0f 0xac 0x04
[   40.226667] RTW: 0x01 0x00 0x00 0x0f 0xac 0x04 0x01 0x00
[   40.227407] RTW: 0x00 0x0f 0xac 0x02 0x00 0x00 0xff 0xff
[   40.228565] RTW: rtw_wx_set_freq: set to channel 6
[   40.256926] RTW: set bssid:64:e5:99:7f:3a:f2
[   40.257534] RTW: Set BSSID under fw_state=0x00000008
[   40.258241] RTW: [by_bssid:1][assoc_ssid:][to_roam:0] new candidate: TODDHWANG(64:e5:99:7f:3a:f2, ch6) rssi:-43
[   40.259622] RTW: rtw_select_and_join_from_scanned_queue: candidate: TODDHWANG(64:e5:99:7f:3a:f2, ch:6)
[   40.260884] RTW: link to Broadcom AP
[   40.261395] RTW: [HT] HAL Support STBC = 0x01
[   40.262004] RTW: [HT] Declare supporting RX STBC = 1
[   40.262673] RTW: rtw_restructure_ht_ie IEEE80211_HT_CAP_MAX_AMSDU is set
[   40.263626] RTW: =>rtw_wx_set_essid
[   40.264147] RTW: ssid=TODDHWANG, len=9
[   40.264690] RTW: set ssid [TODDHWANG] fw_state=0x00000088
[   40.264749] RTW: rtw_chk_start_clnt_join(wlan1) req: 6,0,0
[   40.265435] RTW: Set SSID under fw_state=0x00000088
[   40.265449] RTW: <=rtw_wx_set_essid, ret 0
[   40.266189] RTW: rtw_chk_start_clnt_join(wlan1) union: 6,0,0
**[   40.266200] ------------[ cut here ]------------
[   40.268986] WARNING: CPU: 1 PID: 214 at net/wireless/nl80211.c:15503 cfg80211_ch_switch_notify+0xac/0xc0 [cfg80211]
[   40.270362] Modules linked in: nft_masq_ipv4 nft_masq nft_chain_nat_ipv4 nf_nat_ipv4 nf_nat nf_conntrack nf_defrag_ipv4 nf_defrag_ipv6 8192eu(O) mac80211 bridge brcmfmac cfg80211 brcmutil
[   40.272596] CPU: 1 PID: 214 Comm: RTW_CMD_THREAD Tainted: G           O      4.19.127 #10
[   40.273675] Hardware name: Raspberry Pi 3 Model B Rev 1.2 (DT)
[   40.274453] pstate: 40000005 (nZcv daif -PAN -UAO)
[   40.275167] pc : cfg80211_ch_switch_notify+0xac/0xc0 [cfg80211]
[   40.276025] lr : cfg80211_ch_switch_notify+0xac/0xc0 [cfg80211]
[   40.276810] sp : ffffff8008f9bc50
[   40.277254] x29: ffffff8008f9bc50 x28: ffffffe2b50c8f80
[   40.277965] x27: ffffff8008af1b08 x26: 0000000000000001
[   40.278676] x25: ffffffe2b425bae0 x24: 0000000000000000
[   40.279386] x23: 0000000000000000 x22: ffffff8008af1000
[   40.280096] x21: 0000000000000985 x20: 0000000000000985
[   40.280807] x19: ffffffa083e98608 x18: 0000000000000010
[   40.281517] x17: 0000000000000000 x16: 0000000000000000
[   40.282228] x15: ffffffffffffffff x14: ffffffa083e98608
[   40.282938] x13: ffffffa103f0d90f x12: ffffffa083f0d917
[   40.283648] x11: ffffffa083ea8000 x10: ffffff8008f9b980
[   40.284358] x9 : 00000000ffffffd0 x8 : 2d2d2d2d2d2d2d2d
[   40.285068] x7 : 5d20657265682074 x6 : ffffffe2bab959e0
[   40.285779] x5 : ffffffe2bab959e0 x4 : 0000004236de1000
[   40.286490] x3 : 0000000000000001 x2 : d1f0dda3ba149d00
[   40.287200] x1 : 0000000000000000 x0 : 0000000000000024
[   40.287910] Call trace:
[   40.288315]  cfg80211_ch_switch_notify+0xac/0xc0 [cfg80211]
[   40.289354]  rtw_cfg80211_ch_switch_notify+0xf0/0x140 [8192eu]
[   40.290354]  rtw_chk_start_clnt_join+0x7c/0x170 [8192eu]
[   40.291273]  join_cmd_hdl+0x1ec/0x330 [8192eu]
[   40.292060]  rtw_cmd_thread+0x324/0x590 [8192eu]
[   40.292684]  kthread+0x124/0x130
[   40.293123]  ret_from_fork+0x10/0x1c
[   40.293602] ---[ end trace 82ab4548ce8c4f59 ]---**
[   40.295513] RTW: rtw_hal_set_bssid wlan1- hw port -0 BSSID: 64:e5:99:7f:3a:f2
[   40.774748] RTW: IEEE 802.11 element parse ignored unknown element (id=47 elen=1)
[   40.775770] RTW: unknown vendor specific information element ignored (vendor OUI 00:10:18 len=9)
[   40.776948] RTW: OnBeacon: beacon keys ready
[   40.777520] RTW: link to Broadcom AP
[   40.778004] RTW: start auth
[   40.778389] RTW: issue_auth
[   40.781655] RTW: OnAuthClient
[   40.782080] RTW: auth success, start assoc
[   40.782639] RTW: network.SupportedRates[0]=82
[   40.783255] RTW: network.SupportedRates[1]=84
[   40.783847] RTW: network.SupportedRates[2]=0B
[   40.784459] RTW: network.SupportedRates[3]=16
[   40.785058] RTW: network.SupportedRates[4]=24
[   40.785647] RTW: network.SupportedRates[5]=30
[   40.786260] RTW: network.SupportedRates[6]=48
[   40.786849] RTW: network.SupportedRates[7]=6C
[   40.787451] RTW: network.SupportedRates[8]=0C
[   40.788053] RTW: network.SupportedRates[9]=12
[   40.788638] RTW: network.SupportedRates[10]=18
[   40.789262] RTW: network.SupportedRates[11]=60
[   40.789863] RTW: bssrate_len = 12
[   40.794354] RTW: OnAssocRsp
[   40.794772] RTW: report_join_res(1, 0)
[   40.795298] RTW: rtw_joinbss_update_network
[   40.795881] RTW: +rtw_update_ht_cap()
[   40.796418] RTW: rtw_alloc_macid(wlan1) if1, mac_addr:64:e5:99:7f:3a:f2 macid:0
[   40.797411] RTW: rtw_joinbss_update_stainfo
[   40.798005] RTW: rtw_get_sta_tx_nss: 2 SS
[   40.798580] RTW: rtw_get_sta_rx_nss: 2 SS
[   40.799134] RTW: STA - MAC_ID:0, Tx - 2 SS, Rx - 2 SS
[   40.799821] RTW: STA - MAC_ID:0, SM_PS 3
[   40.800367] RTW: ht_mcs_set=ff ff 00 00, nss=2, bitmap=0000ffff
[   40.801168] RTW: ### Set STA_(0) info ###
[   40.801728] RTW: rtw_cfg80211_indicate_connect(wlan1)
[   40.802459] RTW: assoc success

Please post the output of `uname -r` and `lsmod` (before connecting to a network)

-------- Mensagem Original --------
Ativo 15 de jul de 2020 5:14 PM, ToddHwang-sw escreveu:

> [WLAN] RTK Loading modules...
> [ 25.968575] 8192eu: loading out-of-tree module taints kernel.
> [ 26.312268] RTW: module init start
> [ 26.312746] RTW: rtl8192eu v5.6.4_35685.20191108_COEX20171113-0047
> [ 26.313616] RTW: build time: Jul 15 2020 13:01:24
> [ 26.314268] RTW: rtl8192eu BT-Coex version = COEX20171113-0047
> [ 26.315140] RTW: rtw_inetaddr_notifier_register
> [ 26.315955] RTW:
> [ 26.315955] usb_endpoint_descriptor(0):
> [ 26.316763] RTW: bLength=7
> [ 26.317154] RTW: bDescriptorType=5
> [ 26.317610] RTW: bEndpointAddress=84
> [ 26.318128] RTW: wMaxPacketSize=512
> [ 26.318605] RTW: bInterval=0
> [ 26.319028] RTW: RT_usb_endpoint_is_bulk_in = 4
> [ 26.319639] RTW:
> [ 26.319639] usb_endpoint_descriptor(1):
> [ 26.320442] RTW: bLength=7
> [ 26.320808] RTW: bDescriptorType=5
> [ 26.321297] RTW: bEndpointAddress=5
> [ 26.321771] RTW: wMaxPacketSize=512
> [ 26.322266] RTW: bInterval=0
> [ 26.322662] RTW: RT_usb_endpoint_is_bulk_out = 5
> [ 26.323311] RTW:
> [ 26.323311] usb_endpoint_descriptor(2):
> [ 26.324122] RTW: bLength=7
> [ 26.324493] RTW: bDescriptorType=5
> [ 26.324953] RTW: bEndpointAddress=6
> [ 26.325457] RTW: wMaxPacketSize=512
> [ 26.325933] RTW: bInterval=0
> [ 26.326350] RTW: RT_usb_endpoint_is_bulk_out = 6
> [ 26.326974] RTW:
> [ 26.326974] usb_endpoint_descriptor(3):
> [ 26.327771] RTW: bLength=7
> [ 26.328159] RTW: bDescriptorType=5
> [ 26.328625] RTW: bEndpointAddress=87
> [ 26.329138] RTW: wMaxPacketSize=64
> [ 26.329600] RTW: bInterval=3
> [ 26.330014] RTW: RT_usb_endpoint_is_int_in = 7, Interval = 3
> [ 26.330781] RTW:
> [ 26.330781] usb_endpoint_descriptor(4):
> [ 26.331597] RTW: bLength=7
> [ 26.331969] RTW: bDescriptorType=5
> [ 26.332457] RTW: bEndpointAddress=8
> [ 26.332930] RTW: wMaxPacketSize=512
> [ 26.333429] RTW: bInterval=0
> [ 26.333818] RTW: RT_usb_endpoint_is_bulk_out = 8
> [ 26.334468] RTW: nr_endpoint=5, in_num=2, out_num=3
> [ 26.334468]
> [ 26.335335] RTW: USB_SPEED_HIGH
> [ 26.335759] RTW: CHIP TYPE: RTL8192E
> [ 26.336622] RTW: read_chip_version_8192e 0xF0 = 0xc441135
> [ 26.337375] RTW: rtw_hal_config_rftype RF_Type is 2 TotalTxPath is 2
> [ 26.338259] RTW: Chip Version Info: CHIP_8192E_Normal_Chip_SMIC_B_CUT_2T2R_RomVer(0)
> [ 26.339311] RTW: _ConfigChipOutEP_8192E OutEpQueueSel(0x07), OutEpNumber(3)
> [ 26.340492] RTW: Boot from EFUSE, Autoload OK !
> [ 26.477199] RTW: HW EFUSE
> [ 26.477569] RTW: 0x000: 29 81 00 7C 01 40 03 00 40 74 04 50 14 00 00 00
> [ 26.478592] RTW: 0x010: 27 27 26 26 26 26 2A 2A 29 29 29 F2 EF EF FF FF
> [ 26.479597] RTW: 0x020: FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF
> [ 26.480610] RTW: 0x030: FF FF FF FF FF FF FF FF FF FF 2A 2A 28 28 28 28
> [ 26.481610] RTW: 0x040: 2B 2B 2A 29 29 F2 EF EF FF FF FF FF FF FF FF FF
> [ 26.482624] RTW: 0x050: FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF
> [ 26.483645] RTW: 0x060: FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF
> [ 26.484657] RTW: 0x070: FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF
> [ 26.485655] RTW: 0x080: FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF
> [ 26.486669] RTW: 0x090: FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF
> [ 26.487666] RTW: 0x0A0: FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF
> [ 26.488675] RTW: 0x0B0: FF FF FF FF FF FF FF FF A2 1F 20 00 00 00 FF FF
> [ 26.489673] RTW: 0x0C0: FF 01 00 10 00 00 00 FF 00 00 FF FF FF FF FF FF
> [ 26.490684] RTW: 0x0D0: 57 23 09 01 E7 47 02 D4 6E 0E 13 B9 8C 0A 03 52
> [ 26.491700] RTW: 0x0E0: 65 61 6C 74 65 6B 20 0E 03 38 30 32 2E 31 31 6E
> [ 26.492713] RTW: 0x0F0: 20 4E 49 43 20 00 00 FF FF FF FF FF FF FF FF FF
> [ 26.493711] RTW: 0x100: FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF
> [ 26.494723] RTW: 0x110: FF FF FF FF FF FF FF 0D 03 00 05 00 30 00 00 00
> [ 26.495721] RTW: 0x120: 00 93 FF FF FF FF FF FF FF FF FF FF FF FF FF FF
> [ 26.496730] RTW: 0x130: FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF
> [ 26.497728] RTW: 0x140: FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF
> [ 26.498739] RTW: 0x150: FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF
> [ 26.499762] RTW: 0x160: FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF
> [ 26.500773] RTW: 0x170: FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF
> [ 26.501771] RTW: 0x180: FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF
> [ 26.502784] RTW: 0x190: FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF
> [ 26.503784] RTW: 0x1A0: FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF
> [ 26.504792] RTW: 0x1B0: FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF
> [ 26.505789] RTW: 0x1C0: FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF
> [ 26.506801] RTW: 0x1D0: FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF
> [ 26.507819] RTW: 0x1E0: FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF
> [ 26.508831] RTW: 0x1F0: FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF
> [ 26.509833] RTW: EEPROM ID=0x8129
> [ 26.510313] RTW: VID = 0x2357, PID = 0x0109
> [ 26.510881] RTW: Customer ID: 0x00, SubCustomer ID: 0xCD
> [ 26.511626] RTW: Hal_ReadPowerSavingMode8192E...bHWPwrPindetect(0)-bHWPowerdown(0) ,bSupportRemoteWakeup(1)
> [ 26.512936] RTW: ### PS params=> power_mgnt(2),usbss_enable(0) ###
> [ 26.513813] RTW: EEPROMRegulatory = 0x1
> [ 26.514349] RTW: Board Type: Dongle or WIFI only Module
> [ 26.515080] RTW: Hal_EfuseParseBTCoexistInfo8192E: Disable BT-coex, wifi ant_num=2
> [ 26.516126] RTW: hal_com_config_channel_plan chplan:0x22
> [ 26.516838] RTW: crystal_cap: 0x1f
> [ 26.517330] RTW: ThermalMeter = 0x20
> [ 26.517818] RTW: SWAS: bHwAntDiv = 0, TRxAntDivType = 0
> [ 26.518546] RTW: pHalData->ExternalPA_2G = 0 , pHalData->ExternalLNA_2G = 0
> [ 26.519490] RTW: pHalData->TypeGLNA is 0x0
> [ 26.520071] RTW: pHalData->rfe_type is 0xff
> [ 26.522653] RTW: kfree flag:0
> [ 26.523080] RTW: rtw_hal_read_chip_info in 183 ms
> [ 26.523751] RTW: init_channel_set((null)) ChannelPlan ID:0x22, ch num:11
> [ 26.524861] RTW: init_mlme_default_rate_set: support CCK
> [ 26.525592] RTW: init_mlme_default_rate_set: support OFDM
> [ 26.527053] RTW: NR_RECVBUFF: 8
> [ 26.527484] RTW: MAX_RECVBUF_SZ: 32768
> [ 26.528031] RTW: NR_PREALLOC_RECV_SKB: 8
> [ 26.528943] RTW: rtw_alloc_macid((null)) if1, mac_addr:ff:ff:ff:ff:ff:ff macid:1
> [ 26.530001] RTW: IQK FW offload:disable
> [ 26.530532] RTW: rtw_regsty_chk_target_tx_power_valid return _FALSE for band:0, path:0, rs:0, t:-1
> [ 26.531782] RTW: pwrctrlpriv.bSupportRemoteWakeup~~~~~~
> [ 26.532500] RTW: pwrctrlpriv.bSupportRemoteWakeup~~~[1]~~~
> [ 26.533267] RTW: can't get autopm:
> [ 26.533737] RTW: rtw_macaddr_cfg mac addr:d4:6e:0e:13:b9:8c
> [ 26.534513] RTW: bDriverStopped:True, bSurpriseRemoved:False, bup:0, hw_init_completed:0
> [ 26.535664] RTW: rtw_wiphy_alloc(phy1)
> [ 26.536207] RTW: rtw_wdev_alloc(padapter=0000000024465bc4)
> [ 26.536950] RTW: rtw_wiphy_register(phy1)
> [ 26.537519] RTW: Register RTW cfg80211 vendor cmd(0x67) interface
> [ 26.538547] RTW: rtw_reg_notifier: NL80211_REGDOM_SET_BY_USER alpha2:US
> [ 26.539464] RTW: rtw_set_country(): not applied
> [ 26.544820] RTW: rtw_ndev_init(wlan1) if1 mac_addr=d4:6e:0e:13:b9:8c
> [ 26.546192] RTW: rtw_ndev_notifier_call(wlan1) state:16
> [ 26.548079] RTW: cfg80211_rtw_get_txpower
> [ 26.548638] RTW: rtw_ndev_notifier_call(wlan1) state:5
> [ 26.549634] usbcore: registered new interface driver rtl8192eu
> [ 26.550450] RTW: module init ret=0
> [ 26.569030] RTW: rtw_ndev_notifier_call(wlan1) state:13
> [ 26.569749] RTW: _netdev_open(wlan1) , bup=0
> [ 26.570372] RTW: ==>_InitPowerOn_8192EU
> [ 26.574243] RTW: SetHwReg: bMacPwrCtrlOn=1
> [ 26.575471] RTW: ===> FirmwareDownload88E() fw source from Header.
> [ 26.576333] RTW: FirmwareDownload8192E fw:NIC, size: 32286
> [ 26.577106] RTW: FirmwareDownload8192E: fw_ver=35 fw_subver=7 sig=0x92e1
> [ 26.603561] RTW: polling_fwdl_chksum: Checksum report OK! (1, 0ms), REG_MCUFWDL:0x00070305
> [ 26.606386] RTW: =====> _8051Reset8192E(): 8051 reset success .
> [ 26.634702] RTW: _FWFreeToGo8192E: Polling FW ready OK! (245, 30ms), REG_MCUFWDL:0x000706c6
> [ 26.635859] RTW: FWDL success. write_fw:1, 57ms
> [ 27.251799] RTW: Auto InitLLTTable8192E success
> [ 27.419137] RTW: phy_SpurCalibration_8192E =>AFE_PHASE_SEL
> [ 27.484110] RTW: ===FixSpur NOT Pass!
> [ 27.883945] RTW: rtl8192eu_hal_init in 1313ms
> [ 27.884564] RTW: rtw_hal_set_macaddr_port wlan1- hw port(0) mac_addr =d4:6e:0e:13:b9:8c
> [ 27.887043] RTW: rtw_hal_get_macaddr_port wlan1- hw port(0) mac_addr =d4:6e:0e:13:b9:8c
> [ 27.888138] RTW: wlan1- hw port(0) mac_addr =d4:6e:0e:13:b9:8c
> [ 27.889296] RTW: [HW_VAR_ENABLE_RX_BAR] 0x6A2=0x100
> [ 27.907004] RTW: wlan1- Port-0 set opmode = 2
> [ 27.907629] RTW: rtw_hal_set_macaddr_port wlan1- hw port(0) mac_addr =d4:6e:0e:13:b9:8c
> [ 27.910442] RTW: MAC Address = d4:6e:0e:13:b9:8c
> [ 27.911090] RTW: rtw_start_drv_threads(wlan1) enter
> [ 27.911747] RTW: rtw_start_drv_threads(wlan1) start RTW_CMD_THREAD
> [ 27.912958] RTW: rtw_cfg80211_init_wiphy:rf_type=2
> [ 27.913629] RTW: [HT] HAL Support STBC = 0x01
> [ 27.914248] RTW: -871x_drv - drv_open, bup=1
> [ 27.915162] IPv6: ADDRCONF(NETDEV_UP): wlan1: link is not ready
> [ 27.916016] RTW: cfg80211_rtw_set_power_mgmt(wlan1) enabled:1, timeout:-1
> [ 27.916927] RTW: rtw_ndev_notifier_call(wlan1) state:1
> [WLAN] Building up /var/tmp/wpa_supplicant/1.conf
> [WLAN] Running WPA Suppplicant !!
> Successfully initialized wpa_supplicant
> [ 28.085243] RTW: [rtw_wx_set_pmkid] IW_PMKSA_FLUSH!
> [ 28.085935] RTW: rtw_ps_deny_cancel(wlan1): [ERROR] Reason 7 had been canceled before!!
> [ 28.087047] RTW: set_mode = IW_MODE_INFRA
> [ 28.087634] RTW: wlan1- Port-0 set opmode = 2
> [ 28.088285] RTW: rtw_hal_set_macaddr_port wlan1- hw port(0) mac_addr =d4:6e:0e:13:b9:8c
> [ 28.092064] RTW: set bssid:00:00:00:00:00:00
> ioctl[SI[ 28.092722] RTW: =>rtw_wx_set_essid
> OCSIWAP][ 28.093319] RTW: <=rtw_wx_set_essid, ret 0
> : Operation not permitted
> [ 29.109514] RTW: rtw_ndev_notifier_call(wlan1) state:4
> [ 29.120211] RTW: [rtw_wx_set_pmkid] IW_PMKSA_FLUSH!
> [ 29.247141] RTW: rtw_wx_set_scan(wlan1)
> [ 29.248253] RTW: wlan1 sleep m0=0x00000002, ori reg_0x4d4=0x00000000
> udhcpc: started, v1.28.0
> Waiting IP .. 0
> Setting IP address 0.0.0.0 on wlan1
> udhcpc: sending discover
> [ 29.922547] RTW: _rtw_phydm_pwr_tracking_rate_check i=0 tx_rate =0x2
> [ 29.923433] RTW: _rtw_phydm_pwr_tracking_rate_check tx_low_rate (unlinked to any AP)=0x2
> Waiting IP .. 1
> [ 30.598461] RTW: wlan1 wakeup m0=0x00000002, ori reg_0x4d4=0x00000002
> [ 30.599514] RTW: survey done event(33) band:0 for wlan1
> [ 30.600368] RTW: rtw_indicate_scan_done(wlan1)
> [ 30.602019] RTW: ==>rtw_ps_processor .fw_state(8)
> [ 30.602654] RTW: ==>ips_enter cnts:1
> [ 30.603169] RTW: nolinked power save enter
> [ 30.603719] RTW: ===> rtw_ips_pwr_down...................
> [ 30.604473] RTW: ====> rtw_ips_dev_unload...
> [ 30.632057] RTW: usb_read_port_cancel
> [ 30.632572] RTW: usb_read_port_complete() RX Warning! bDriverStopped(False) OR bSurpriseRemoved(False)
> [ 30.633848] RTW: usb_read_port_complete() RX Warning! bDriverStopped(False) OR bSurpriseRemoved(False)
> [ 30.635124] RTW: usb_read_port_complete() RX Warning! bDriverStopped(False) OR bSurpriseRemoved(False)
> [ 30.636419] RTW: usb_read_port_complete() RX Warning! bDriverStopped(False) OR bSurpriseRemoved(False)
> [ 30.637717] RTW: usb_read_port_complete() RX Warning! bDriverStopped(False) OR bSurpriseRemoved(False)
> [ 30.639008] RTW: usb_read_port_complete() RX Warning! bDriverStopped(False) OR bSurpriseRemoved(False)
> [ 30.640284] RTW: usb_read_port_complete() RX Warning! bDriverStopped(False) OR bSurpriseRemoved(False)
> [ 30.641558] RTW: usb_read_port_complete() RX Warning! bDriverStopped(False) OR bSurpriseRemoved(False)
> [ 30.642823] RTW: usb_write_port_cancel
> [ 30.643391] RTW: ==> rtl8192eu_hal_deinit
> [ 30.644635] RTW: bkeepfwalive(0)
> [ 30.645100] RTW: hal_poweroff_8192eu
> [ 30.649432] RTW: SetHwReg: bMacPwrCtrlOn=0
> [ 30.650009] RTW: <=== rtw_ips_pwr_down..................... in 47ms
> ...
> ...
> [ 39.253477] RTW: _rtw_pwr_wakeup call ips_leave....
> [ 39.254173] RTW: ==>ips_leave cnts:1
> [ 39.254657] RTW: ===> rtw_ips_pwr_up..............
> [ 39.255361] RTW: ===> ips_netdrv_open.........
> [ 39.255968] RTW: ==>_InitPowerOn_8192EU
> [ 39.259809] RTW: SetHwReg: bMacPwrCtrlOn=1
> [ 39.260947] RTW: ===> FirmwareDownload88E() fw source from Header.
> [ 39.261822] RTW: FirmwareDownload8192E fw:NIC, size: 32286
> [ 39.262582] RTW: FirmwareDownload8192E: fw_ver=35 fw_subver=7 sig=0x92e1
> [ 39.289190] RTW: polling_fwdl_chksum: Checksum report OK! (1, 0ms), REG_MCUFWDL:0x00070605
> [ 39.292005] RTW: =====> _8051Reset8192E(): 8051 reset success .
> [ 39.320363] RTW: FWFreeToGo8192E: Polling FW ready OK! (243, 30ms), REG_MCUFWDL:0x000706c6
> [ 39.321502] RTW: FWDL success. write_fw:1, 57ms
> [ 39.938797] RTW: Auto InitLLTTable8192E success
> [ 40.101796] RTW: phy_SpurCalibration_8192E =>AFE_PHASE_SEL
> [ 40.166112] RTW: === PathA=1, PathB=1
> [ 40.166611] RTW: ===FixSpur Pass!
> [ 40.188464] RTW: rtl8192eu_hal_init in 933ms
> [ 40.189077] RTW: rtw_hal_set_macaddr_port wlan1- hw port(0) mac_addr =d4:6e:0e:13:b9:8c
> [ 40.191544] RTW: rtw_hal_get_macaddr_port wlan1- hw port(0) mac_addr =d4:6e:0e:13:b9:8c
> [ 40.192643] RTW: wlan1- hw port(0) mac_addr =d4:6e:0e:13:b9:8c
> [ 40.193816] RTW: [HW_VAR_ENABLE_RX_BAR] 0x6A2=0x100
> [ 40.211567] RTW: wlan1- Port-0 set opmode = 2
> [ 40.212200] RTW: rtw_hal_set_macaddr_port wlan1- hw port(0) mac_addr =d4:6e:0e:13:b9:8c
> [ 40.215122] RTW: <=== rtw_ips_pwr_up.............. in 961ms
> [ 40.215882] RTW: nolinked power save leave
> [ 40.216626] RTW: ==> ips_leave.....LED(0x0f628282)...
> [ 40.217336] RTW: =>rtw_wx_set_essid
> [ 40.217813] RTW: <=rtw_wx_set_essid, ret 0
> [ 40.218530] RTW: wpa_set_auth_algs, AUTH_ALG_OPEN_SYSTEM
> [ 40.219285] RTW: rtw_ps_deny_cancel(wlan1): [ERROR] Reason 7 had been canceled before!!
> [ 40.220383] RTW: set_mode = IW_MODE_INFRA
> [ 40.220966] RTW: wlan1- Port-0 set opmode = 2
> [ 40.221600] RTW: rtw_hal_set_macaddr_port wlan1- hw port(0) mac_addr =d4:6e:0e:13:b9:8c
> [ 40.225230] RTW:
> [ 40.225230] wpa_ie(length:22):
> [ 40.225919] RTW: 0x30 0x14 0x01 0x00 0x00 0x0f 0xac 0x04
> [ 40.226667] RTW: 0x01 0x00 0x00 0x0f 0xac 0x04 0x01 0x00
> [ 40.227407] RTW: 0x00 0x0f 0xac 0x02 0x00 0x00 0xff 0xff
> [ 40.228565] RTW: rtw_wx_set_freq: set to channel 6
> [ 40.256926] RTW: set bssid:64:e5:99:7f:3a:f2
> [ 40.257534] RTW: Set BSSID under fw_state=0x00000008
> [ 40.258241] RTW: [by_bssid:1][assoc_ssid:][to_roam:0] new candidate: TODDHWANG(64:e5:99:7f:3a:f2, ch6) rssi:-43
> [ 40.259622] RTW: rtw_select_and_join_from_scanned_queue: candidate: TODDHWANG(64:e5:99:7f:3a:f2, ch:6)
> [ 40.260884] RTW: link to Broadcom AP
> [ 40.261395] RTW: [HT] HAL Support STBC = 0x01
> [ 40.262004] RTW: [HT] Declare supporting RX STBC = 1
> [ 40.262673] RTW: rtw_restructure_ht_ie IEEE80211_HT_CAP_MAX_AMSDU is set
> [ 40.263626] RTW: =>rtw_wx_set_essid
> [ 40.264147] RTW: ssid=TODDHWANG, len=9
> [ 40.264690] RTW: set ssid [TODDHWANG] fw_state=0x00000088
> [ 40.264749] RTW: rtw_chk_start_clnt_join(wlan1) req: 6,0,0
> [ 40.265435] RTW: Set SSID under fw_state=0x00000088
> [ 40.265449] RTW: <=rtw_wx_set_essid, ret 0
> [ 40.266189] RTW: rtw_chk_start_clnt_join(wlan1) union: 6,0,0
> [ 40.266200] ------------[ cut here ]------------
> [ 40.268986] WARNING: CPU: 1 PID: 214 at net/wireless/nl80211.c:15503 cfg80211_ch_switch_notify+0xac/0xc0 [cfg80211]
> [ 40.270362] Modules linked in: nft_masq_ipv4 nft_masq nft_chain_nat_ipv4 nf_nat_ipv4 nf_nat nf_conntrack nf_defrag_ipv4 nf_defrag_ipv6 8192eu(O) mac80211 bridge brcmfmac cfg80211 brcmutil
> [ 40.272596] CPU: 1 PID: 214 Comm: RTW_CMD_THREAD Tainted: G O 4.19.127 [#10](https://github.com/Mange/rtl8192eu-linux-driver/issues/10)
> [ 40.273675] Hardware name: Raspberry Pi 3 Model B Rev 1.2 (DT)
> [ 40.274453] pstate: 40000005 (nZcv daif -PAN -UAO)
> [ 40.275167] pc : cfg80211_ch_switch_notify+0xac/0xc0 [cfg80211]
> [ 40.276025] lr : cfg80211_ch_switch_notify+0xac/0xc0 [cfg80211]
> [ 40.276810] sp : ffffff8008f9bc50
> [ 40.277254] x29: ffffff8008f9bc50 x28: ffffffe2b50c8f80
> [ 40.277965] x27: ffffff8008af1b08 x26: 0000000000000001
> [ 40.278676] x25: ffffffe2b425bae0 x24: 0000000000000000
> [ 40.279386] x23: 0000000000000000 x22: ffffff8008af1000
> [ 40.280096] x21: 0000000000000985 x20: 0000000000000985
> [ 40.280807] x19: ffffffa083e98608 x18: 0000000000000010
> [ 40.281517] x17: 0000000000000000 x16: 0000000000000000
> [ 40.282228] x15: ffffffffffffffff x14: ffffffa083e98608
> [ 40.282938] x13: ffffffa103f0d90f x12: ffffffa083f0d917
> [ 40.283648] x11: ffffffa083ea8000 x10: ffffff8008f9b980
> [ 40.284358] x9 : 00000000ffffffd0 x8 : 2d2d2d2d2d2d2d2d
> [ 40.285068] x7 : 5d20657265682074 x6 : ffffffe2bab959e0
> [ 40.285779] x5 : ffffffe2bab959e0 x4 : 0000004236de1000
> [ 40.286490] x3 : 0000000000000001 x2 : d1f0dda3ba149d00
> [ 40.287200] x1 : 0000000000000000 x0 : 0000000000000024
> [ 40.287910] Call trace:
> [ 40.288315] cfg80211_ch_switch_notify+0xac/0xc0 [cfg80211]
> [ 40.289354] rtw_cfg80211_ch_switch_notify+0xf0/0x140 [8192eu]
> [ 40.290354] rtw_chk_start_clnt_join+0x7c/0x170 [8192eu]
> [ 40.291273] join_cmd_hdl+0x1ec/0x330 [8192eu]
> [ 40.292060] rtw_cmd_thread+0x324/0x590 [8192eu]
> [ 40.292684] kthread+0x124/0x130
> [ 40.293123] ret_from_fork+0x10/0x1c
> [ 40.293602] ---[ end trace 82ab4548ce8c4f59 ]---
> [ 40.295513] RTW: rtw_hal_set_bssid wlan1- hw port -0 BSSID: 64:e5:99:7f:3a:f2
> [ 40.774748] RTW: IEEE 802.11 element parse ignored unknown element (id=47 elen=1)
> [ 40.775770] RTW: unknown vendor specific information element ignored (vendor OUI 00:10:18 len=9)
> [ 40.776948] RTW: OnBeacon: beacon keys ready
> [ 40.777520] RTW: link to Broadcom AP
> [ 40.778004] RTW: start auth
> [ 40.778389] RTW: issue_auth
> [ 40.781655] RTW: OnAuthClient
> [ 40.782080] RTW: auth success, start assoc
> [ 40.782639] RTW: network.SupportedRates[0]=82
> [ 40.783255] RTW: network.SupportedRates[1]=84
> [ 40.783847] RTW: network.SupportedRates[2]=0B
> [ 40.784459] RTW: network.SupportedRates[3]=16
> [ 40.785058] RTW: network.SupportedRates[4]=24
> [ 40.785647] RTW: network.SupportedRates[5]=30
> [ 40.786260] RTW: network.SupportedRates[6]=48
> [ 40.786849] RTW: network.SupportedRates[7]=6C
> [ 40.787451] RTW: network.SupportedRates[8]=0C
> [ 40.788053] RTW: network.SupportedRates[9]=12
> [ 40.788638] RTW: network.SupportedRates[10]=18
> [ 40.789262] RTW: network.SupportedRates[11]=60
> [ 40.789863] RTW: bssrate_len = 12
> [ 40.794354] RTW: OnAssocRsp
> [ 40.794772] RTW: report_join_res(1, 0)
> [ 40.795298] RTW: rtw_joinbss_update_network
> [ 40.795881] RTW: +rtw_update_ht_cap()
> [ 40.796418] RTW: rtw_alloc_macid(wlan1) if1, mac_addr:64:e5:99:7f:3a:f2 macid:0
> [ 40.797411] RTW: rtw_joinbss_update_stainfo
> [ 40.798005] RTW: rtw_get_sta_tx_nss: 2 SS
> [ 40.798580] RTW: rtw_get_sta_rx_nss: 2 SS
> [ 40.799134] RTW: STA - MAC_ID:0, Tx - 2 SS, Rx - 2 SS
> [ 40.799821] RTW: STA - MAC_ID:0, SM_PS 3
> [ 40.800367] RTW: ht_mcs_set=ff ff 00 00, nss=2, bitmap=0000ffff
> [ 40.801168] RTW: ### Set STA(0) info ###
> [ 40.801728] RTW: rtw_cfg80211_indicate_connect(wlan1)
> [ 40.802459] RTW: assoc success
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, [view it on GitHub](https://github.com/Mange/rtl8192eu-linux-driver/issues/188), or [unsubscribe](https://github.com/notifications/unsubscribe-auth/AN2H5ZAMGY5QERCOR3EPWSLR3YEYTANCNFSM4O246MDA).

--- a/core/rtw_ap.c
+++ b/core/rtw_ap.c
@@ -1784,7 +1784,7 @@ chbw_decision:
 				, pdvobj->padapters[i]->mlmeextpriv.cur_channel
 				, pdvobj->padapters[i]->mlmeextpriv.cur_bwmode
 				, pdvobj->padapters[i]->mlmeextpriv.cur_ch_offset
-				, ht_option);
+				, ht_option, 0);
 		}
 	}
 #endif /* defined(CONFIG_IOCTL_CFG80211) && (LINUX_VERSION_CODE >= KERNEL_VERSION(3, 5, 0)) */

--- a/core/rtw_mlme_ext.c
+++ b/core/rtw_mlme_ext.c
@@ -16096,7 +16096,7 @@ void rtw_join_done_chk_ch(_adapter *adapter, int join_res)
 
 						rtw_cfg80211_ch_switch_notify(iface
 							, mlmeext->cur_channel, mlmeext->cur_bwmode, mlmeext->cur_ch_offset
-							, ht_option);
+							, ht_option, 0);
 						#endif
 					}
 				}
@@ -16314,7 +16314,7 @@ exit:
 				the bss freq is updated by channel switch event.
 			*/
 			rtw_cfg80211_ch_switch_notify(adapter,
-				cur_ch, cur_bw, cur_ch_offset, ht_option);
+				cur_ch, cur_bw, cur_ch_offset, ht_option, 1);
 		}
 #endif
 	}

--- a/os_dep/linux/ioctl_cfg80211.c
+++ b/os_dep/linux/ioctl_cfg80211.c
@@ -424,19 +424,27 @@ bool rtw_cfg80211_allow_ch_switch_notify(_adapter *adapter)
 	return 1;
 }
 
-u8 rtw_cfg80211_ch_switch_notify(_adapter *adapter, u8 ch, u8 bw, u8 offset, u8 ht)
+u8 rtw_cfg80211_ch_switch_notify(_adapter *adapter, u8 ch, u8 bw, u8 offset,
+	u8 ht, bool started)
 {
 	struct wiphy *wiphy = adapter_to_wiphy(adapter);
 	u8 ret = _SUCCESS;
 
 #if (LINUX_VERSION_CODE >= KERNEL_VERSION(3, 8, 0))
-	struct cfg80211_chan_def chdef;
-
-	if (!rtw_cfg80211_allow_ch_switch_notify(adapter))
-		goto exit;
+	struct cfg80211_chan_def chdef = {};
 
 	ret = rtw_chbw_to_cfg80211_chan_def(wiphy, &chdef, ch, bw, offset, ht);
 	if (ret != _SUCCESS)
+		goto exit;
+
+#if (LINUX_VERSION_CODE >= KERNEL_VERSION(3, 19, 0))
+	if (started) {
+		cfg80211_ch_switch_started_notify(adapter->pnetdev, &chdef, 0);
+		goto exit;
+	}
+#endif
+
+	if (!rtw_cfg80211_allow_ch_switch_notify(adapter))
 		goto exit;
 
 	cfg80211_ch_switch_notify(adapter->pnetdev, &chdef);

--- a/os_dep/linux/ioctl_cfg80211.h
+++ b/os_dep/linux/ioctl_cfg80211.h
@@ -407,7 +407,7 @@ void rtw_cfg80211_deinit_rfkill(struct wiphy *wiphy);
 #endif
 
 #if (LINUX_VERSION_CODE >= KERNEL_VERSION(3, 5, 0))
-u8 rtw_cfg80211_ch_switch_notify(_adapter *adapter, u8 ch, u8 bw, u8 offset, u8 ht);
+u8 rtw_cfg80211_ch_switch_notify(_adapter *adapter, u8 ch, u8 bw, u8 offset, u8 ht, bool started);
 #endif
 
 #if (LINUX_VERSION_CODE >= KERNEL_VERSION(2, 6, 26)) && (LINUX_VERSION_CODE < KERNEL_VERSION(4, 7, 0))

